core/chains/evm/client/errors.go

```diff
@@ -48,13 +48,14 @@ const (
 	TerminallyUnderpriced
 	InsufficientEth
 	TxFeeExceedsCap
-	// Note: OptimismFeeTooLow/OptimismFeeTooHigh have a very specific meaning
-	// specific to Optimism and clones. Do not implement this for non-L2
+	// Note: L2FeeTooLow/L2FeeTooHigh/L2Full have a very specific meaning specific
+	// to L2s (Arbitrum, Optimism and clones). Do not implement this for non-L2
 	// chains. This is potentially confusing because some RPC nodes e.g.
 	// Nethermind implement an error called `FeeTooLow` which has distinct
 	// meaning from this one.
-	OptimismFeeTooLow
-	OptimismFeeTooHigh
+	L2FeeTooLow
+	L2FeeTooHigh
+	L2Full
```

Comment:
L2Full seems to me very similar to LimitReached error. The only difference I can think of is whether a standard mempool will drop other txs even if the pool is full, for higher gas prices (Arbitrum might behave the same way as well).